### ISSUE TITLE
Upload files VS Code extension fails due to path escaping

### ISSUE THREAD

**Issue by lohart13** [oracle/netsuite-suitecloud-sdk#744]
### Select the SuiteCloud tool where you encountered the issue

SuiteCloud Extension for VSCode

### Version of the SuiteCloud Tool

v1.4.3

### Operating system

Windows 11 Pro, 21H2

### Node Version / VSCode Version

v20.4.0/1.80.2

### JAVA Version

17.0.7+8-LTS-224

### Bug description

When using the SuiteCloud Extension for Visual Studio Code and trying to update a file to the SuiteScript folder in the file cabinet, VS Code shows the successful upload prompt however looking at the output for VS Code, the file upload fails due to the folder being incorrectly classified as a restricted folder. From the image below, it appears that when the file path is checked, the path has double slashes instead of single which is making the path invalid.

![image](https://github.com/oracle/netsuite-suitecloud-sdk/assets/50547373/163496d1-5269-44dc-88f0-8a931e94c48d)

### Steps To Reproduce

1. Open a valid file to upload in the editor window
2. Press `CTRL+SHIFT+P` and choose the `SuiteCloud: Upload File` option.

### Actual Output

![image](https://github.com/oracle/netsuite-suitecloud-sdk/assets/50547373/163496d1-5269-44dc-88f0-8a931e94c48d)

### Expected Output

The file uploads successfully.

**Comment by ALPHAMUM** [comment]
still no update regarding this issue 🥲, I'm having same error. @lohart13, have you fix yours?

**Comment by alii-gator** [comment]
@ALPHAMUM Are you also on windows?
In case you are still having issues, you can use suitecloud cli while we investigate this! https://www.npmjs.com/package/@oracle/suitecloud-cli

**Comment by ALPHAMUM** [comment]
@alii-gator yes I'm using windows. currently I'm upload my codes manually. I'll check this suitecloud cli thanks.

**Comment by ALPHAMUM** [comment]
the suitecloud CLI works, Thanks @alii-gator

**Comment by tpsteiner** [comment]
Another workaround is to use JDK version 17.0.2

You will have to also install with auto_updates=disabled.

Create a configuration file with the following content:
`AUTO_UPDATE=DisableINSTALL_SILENT=Enable`

Run the Installer with the Configuration File:
`jdk-installer-name.exe INSTALLCFG=path\to\jdk-config.txt`

Verify
`java -version`

**Comment by vantiago** [comment]
@alii-gator Thanks for the SuiteCloud CLI idea...  it works just not as well as SuiteCloud Extension.  Any updates on this issue?  Just making sure I'm not the only one still having this issue.

**Comment by tourajvaziri** [comment]
Hi @vantiago ,
Have you tried the workaround for java 17.0.2?
It is also explained in another ticket:
https://github.com/oracle/netsuite-suitecloud-sdk/issues/711

**Comment by vantiago** [comment]
Thanks @tourajvaziri!  That did the trick!

**Comment by Mohd-Salim05** [comment]
I'm also facing the same issue right now, it is still not fixed

**Comment by lfpvillegas** [comment]
We have released the fix for this issue in last version of the CLI1.9.0 and  VsCode extension 1.6.0
Please check it out and If you are still experiencing this issue, please reopen this ticket.
Thanks
